pkgs/development/ocaml-modules/get-activity/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildDunePackage,
+  get-activity-lib,
+  ppx_expect,
+  cmdliner,
+  dune-build-info,
+  fmt,
+  logs,
+  alcotest
+}:
+
+buildDunePackage rec {
+  pname = "get-activity";
+  inherit (get-activity-lib) version src;
+
+  minimalOCamlVersion = "4.08";
+
+  propagatedBuildInputs = [
```

Comment:
```suggestion
  buildInputs = [
```

Reply:
There was no `META` file, so that got me confused. Thanks!